.env.development

```diff
@@ -47,4 +47,4 @@ INVITE_STUDENTS_EMAIL_TO="someone@domain.com"
 ENABLE_HOME_PAGE_COURSE_API_V2=true
 ENABLE_CHECKLIST_QUALITY=true
 ENABLE_GRADING_METHOD_IN_PROBLEMS=false
-LIBRARY_SUPPORTED_BLOCKS="problem,video,html,drag-and-drop-v2"
+LIBRARY_SUPPORTED_BLOCKS="problem,video,html,drag-and-drop-v2,openassessment,conditional,done,edx_sga,freetextresponse,google-calendar,google-document,invideoquiz,lti_consumer,pdf,poll,survey,word_cloud"
```

Comment:
Once we merge https://github.com/openedx/edx-platform/pull/36292, we can get rid of this `LIBRARY_SUPPORTED_BLOCKS` setting, right?

Reply:
Not yet, `LIBRARY_SUPPORTED_BLOCKS` is used in [canEditComponent](https://github.com/openedx/frontend-app-authoring/blob/06d591df13515b820b219a25ae6fbd3e4d455ffc/src/library-authoring/components/ComponentEditorModal.tsx#L18) and this function is used in places like [ComponentInfo](https://github.com/openedx/frontend-app-authoring/blob/06d591df13515b820b219a25ae6fbd3e4d455ffc/src/library-authoring/component-info/ComponentInfo.tsx#L132) to see if the block is editable. I would need to add support of `useBlockTypesMetadata` in the [context](https://github.com/openedx/frontend-app-authoring/pull/1653#discussion_r1964112763) so that I could use it there as well. What do yo think? Do you think I should make the changes to no longer use LIBRARY_SUPPORTED_BLOCKS?

Reply:
I think as part of completing this "advanced components" epic, we should be able to remove canEditComponent entirely, and just assume that any component is editable if it has been added to a library. Are there any XBlocks that you could copy-paste into a library that definitely won't be editable?

Reply:
Yes, "multi-level" blocks like Problem builds, Step Builder, and Conditional, are not supported in libraries or the new Unit Outline, and can't save in the editor ([ref](https://tasks.opencraft.com/browse/FAL-4030?focusedCommentId=304697&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-304697)). But they are just those blocks. Can we change `LIBRARY_SUPPORTED_BLOCKS` to `LIBRARY_UNSUPPORTED_BLOCKS`?

Reply:
Good idea, I prefer `LIBRARY_UNSUPPORTED_BLOCKS` and if we can also add a comment saying that "multi-level" blocks are unsupported, that would be much more clear.

Reply:
@bradenmacdonald  Updated https://github.com/openedx/frontend-app-authoring/pull/1653/commits/e590e35417b80d0e294e4069b39b6081544110fd

@rpenido Could you test it again?

Reply:
Tested and it is working great @ChrisChV! CC @bradenmacdonald